xcp/dmv.py

```diff
@@ -20,6 +20,29 @@
 # ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
 # SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+"""
+Driver Multiversion (DMV) Management for XenServer/XCP-ng
```

Comment:
We absolutely cannot be making statements about other products.  This needs to be XenServer only.